pkg/storage/note.go

```diff
@@ -49,7 +49,7 @@ func (s *Store) CreateNote(ctx context.Context, note *models.Note) (*uuid.UUID,
 		)
 		return nil, err
 	}
-	return &note.ID, nil
+	return note, nil
```

Comment:
I think per our standards this should return FetchNoteByID(id) rather than just returning note.

Reply:
Yes, as Isaac said. If you don't defensively round-trip from the database, you are only returning the memory copy of the note, and this could hide bugs happening in the INSERT SQL.

Reply:
That makes sense!